src/core/agent/services/identifierService.ts

```diff
@@ -649,6 +663,11 @@ class IdentifierService extends AgentService {
       isPending: res.op.done ? false : true, //this will be updated once the operation is done
       multisigManageAid: identifier.id,
     });
+    if (identifier.groupMetadata) {
```

Comment:
This always has to be set

Reply:
we are using `IdentifierShortDetails` for this and `groupMetadata` is optional in our interface

Reply:
Yes but this will silently do the wrong thing and be very hard to debug. If `groupMetadata` is undefined, but should be defined in this case - throw an error

Reply:
Ok, I will change this.